packages/suite/src/components/suite/modals/ReduxModal/TransactionReviewModal/TransactionReviewModalContent.tsx

```diff
@@ -361,24 +361,28 @@ export const TransactionReviewModalContent = ({
             );
         }
 
-        return (
-            <>
-                <NewModal.Button
-                    isDisabled={!serializedTx}
-                    onClick={handleCopy}
-                    data-testid="@send/copy-raw-transaction"
-                >
-                    <Translation id="COPY_TRANSACTION_TO_CLIPBOARD" />
-                </NewModal.Button>
-                <NewModal.Button
-                    variant="tertiary"
-                    isDisabled={!serializedTx}
-                    onClick={handleDownload}
-                >
-                    <Translation id="DOWNLOAD_TRANSACTION" />
-                </NewModal.Button>
-            </>
-        );
+        if (serializedTx) {
+            return (
+                <>
+                    <NewModal.Button
+                        isDisabled={!serializedTx}
+                        onClick={handleCopy}
+                        data-testid="@send/copy-raw-transaction"
+                    >
+                        <Translation id="COPY_TRANSACTION_TO_CLIPBOARD" />
+                    </NewModal.Button>
+                    <NewModal.Button
+                        variant="tertiary"
+                        isDisabled={!serializedTx}
+                        onClick={handleDownload}
+                    >
+                        <Translation id="DOWNLOAD_TRANSACTION" />
+                    </NewModal.Button>
+                </>
+            );
+        }
+
+        return null;
```

Comment:
was it intentional to return modal for nullable serializedTx? probably not I guess..

Reply:
Before it would always show these buttons, but disabled. We don't want them there in Connect flow and to me it doesn't make sense to have them there if they cannot be used. 
But maybe there is some flow where it makes sense to show them as disabled 🤔